ir/shared/src/main/scala/org/scalajs/ir/Types.scala

```diff
@@ -235,17 +235,18 @@ object Types {
     }
   }
 
-  final val VoidRef = PrimRef(NoType)
-  final val BooleanRef = PrimRef(BooleanType)
-  final val CharRef = PrimRef(CharType)
-  final val ByteRef = PrimRef(ByteType)
-  final val ShortRef = PrimRef(ShortType)
-  final val IntRef = PrimRef(IntType)
-  final val LongRef = PrimRef(LongType)
-  final val FloatRef = PrimRef(FloatType)
-  final val DoubleRef = PrimRef(DoubleType)
-  final val NullRef = PrimRef(NullType)
-  final val NothingRef = PrimRef(NothingType)
+  // @unchecked for the initialization checker of Scala 3
```

Comment:
Could you add to the comment why the init checker fails? (IIUC it is because calling `primRef` before line 249 is problematic).

Reply:
Done.